src/components/Message/utils.tsx

```diff
@@ -306,6 +306,7 @@ export const messageHasReactions = <
 >(
   message?: StreamMessage<StreamChatGenerics>,
 ) => !!message?.latest_reactions && !!message.latest_reactions.length;
+// tktktk
```

Comment:
```suggestion
```

Reply:
@arnautov-anton Looks like this is from an older commit :)